### ISSUE TITLE
Author: http://syriaca.org/person/1319

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-app-data#275]
Nathan, this is probably worth sprucing up and including in the authors if you coudl please.

<tei:div xmlns:tei="http://www.tei-c.org/ns/1.0">
        <idno xmlns="http://www.tei-c.org/ns/1.0" type="URI">http://syriaca.org/person/1319/tei</idno>
        <persName xmlns="http://www.tei-c.org/ns/1.0" xml:id="name1319-2" xml:lang="en">John of
          Qartmin</persName>
        <note xmlns="http://www.tei-c.org/ns/1.0" type="abstract">In hagiography: John of Qartmin or
          John of Beth Sbirino was a noted paleograph who reestablished the use of estrangelo
          writing.</note>
      </tei:div>

**Comment by wlpotter** [comment]
I don't think this record was resolved? It's not marked as an author. http://syriaca.org/person/1319
